bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -73,6 +73,11 @@
             </ul>
           </li>
 
+          <f7-list-item link="/settings/addons/" :title="$t('sidebar.addOnStore')" panel-close :animate="false"
+                        :class="{ currentsection: currentUrl.indexOf('/settings/addons/') >= 0 }">
+            <f7-icon slot="media" ios="f7:bag" aurora="f7:bag" md="material:shopping_bag" color="gray" />
```

Comment:
A minor detail I just noticed too late (it wouldn't have been blocking): in that left side menu the top-level entries' icons (Settings, Developer Tools, About) are "filled" and the sub-levels are "hollow", so in this case the icon should have been e.g. `f7:bag_fill` instead of `f7:bag`; again just a minor design quirk but it contributes to the consistency :sunglasses:

Reply:
Agreed that it should be changed for visual consistency, so here it is: #2053.